pkgs/by-name/je/jellyseerr/package.nix

```diff
@@ -0,0 +1,81 @@
+{
+  lib,
+  pnpm_9,
+  fetchFromGitHub,
+  stdenv,
+  makeWrapper,
+  nodejs,
+  python3,
+  sqlite,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "jellyseerr";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "Fallenbagel";
+    repo = "jellyseerr";
+    rev = "v${version}";
+    hash = "sha256-5kaeqhjUy9Lgx4/uFcGRlAo+ROEOdTWc2m49rq8R8Hs=";
+  };
+
+  pnpmDeps = pnpm_9.fetchDeps {
+    inherit pname version src;
+    hash = "sha256-xu6DeaBArQmnqEnIgjc1DTZujQebSkjuai9tMHeQWCk=";
+  };
+
+  buildInputs = [ sqlite ];
+
+  nativeBuildInputs = [
+    python3
+    nodejs
+    makeWrapper
+    pnpm_9.configHook
+  ];
+
+  preBuild = ''
+    export npm_config_nodedir=${nodejs}
+    pushd node_modules
+    pnpm rebuild bcrypt sqlite3
+    popd
+  '';
+
+  buildPhase = ''
+    runHook preBuild
+    pnpm build
+    pnpm prune --prod --ignore-scripts
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share
+    cp -r -t $out/share .next node_modules dist config package.json
+    runHook postInstall
+  '';
+
+  postInstall = ''
+    mkdir -p $out/bin
+    makeWrapper '${nodejs}/bin/node' "$out/bin/jellyseerr" \
```

Comment:
Instead of setting `WorkingDirectory` on the Systemd service, we could set `--chdir` in [the wrapper script](https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/setup-hooks/make-wrapper.sh#L23). That would have the benefit that the executable can be ran easily on the command line as well.